requirements.txt

```diff
@@ -6,6 +6,7 @@ django-cors-middleware==1.3.1
 django-test-without-migrations==0.4
 djangorestframework==3.5.3
 gunicorn==19.6.0
+pandas==0.19.2
```

Comment:
The problem is that `pip install -r requirements.txt` would fail. And requiring a Anaconda is not common for most webdevs — including the way `conda` conflicts with `virtualenv`…